cmd/common.go

```diff
@@ -173,7 +173,9 @@ func prettyPrint(r *check.Controls, summary check.Summary) {
 			for _, c := range g.Checks {
 				colorPrint(c.State, fmt.Sprintf("%s %s\n", c.ID, c.Text))
 
-				if includeTestOutput && c.State == check.FAIL && len(c.ActualValue) > 0 {
+				if includeTestOutput &&
+					(c.State == check.FAIL || c.State == check.ERRO) &&
```

Comment:
Also WARN should be printed since starting now WARN is a fail just not scored.

Reply:
Did I understand you correctly?

```suggestion
					(c.State == check.FAIL || c.State == check.WARN || c.State == check.ERRO) &&
```